demos/bartlett1932/__init__.py

```diff
@@ -0,0 +1,7 @@
+from dallinger import db
+from experiment import Bartlett1932
+
+
+def build():
+    """Potentially move PsiturkConfig stuff in here"""
+    return Bartlett1932(db.init_db())
```

Comment:
That can go. It was a sketch from early in the process of collectively brainstorming how to pass various things to the experiment constructor. I think there are one or two related tests that should also go.